internal/core/src/storage/FileManager.h

```diff
@@ -27,6 +27,27 @@
 #include "log/Log.h"
 
 namespace milvus::storage {
+enum class FileManagerType { MemFile, DiskFile };
```

Comment:
It appears that the FileManagerType is not being utilized.

Reply:
removed